skype/skype-ps/skype/New-CsOnlineVoicemailPolicy.md

```diff
@@ -136,6 +145,39 @@ Accept pipeline input: False
 Accept wildcard characters: False
 ```
 
+### -PrimarySystemPromptLanguage
+The primary (or first) language that voicemail system prompts will be presented in. Must also set SecondarySystemPromptLanguage. When set, this overrides the user language choice. Please see [Set-CsOnlineVoicemailUserSettings](https://docs.microsoft.com/powershell/module/skype/set-csonlinevoicemailusersettings?view=skype-ps) -PromptLanguage for supported languages.
```

Comment:
```suggestion
The primary (or first) language that voicemail system prompts will be presented in. Must also set SecondarySystemPromptLanguage. When set, this overrides the user language choice. Please see [Set-CsOnlineVoicemailUserSettings](/powershell/module/skype/set-csonlinevoicemailusersettings?view=skype-ps) -PromptLanguage for supported languages.
```

Reply:
Links to articles on docs.microsoft.com must use relative links (/folder/topic-file-name) and not absolute links (https://docs.microsoft.com/folder/topic-file-name).